pages.zh/common/autoconf.md

```diff
@@ -0,0 +1,16 @@
+# 自动配置
+
+> 生成配置脚本，自动配置软件源码包。
+> 更多信息：<https://www.gnu.org/software/autoconf>。
```

Comment:
Using Latin period is a mandatory rule for more info link. See: https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md#zh
```suggestion
> 更多信息：<https://www.gnu.org/software/autoconf>.
```